### PR TITLE
Admin user profile #116

### LINKED ISSUE #116: User Story 21, Admin User Profile Page
```
As an admin user
When I visit a user's profile page ("/admin/users/5")
I see the same information the user would see themselves
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

### PR DESCRIPTION
**Pull request by stoic-plus**
closes #116 

* Adds @user to show route to `admin/users`
  * view is basically the same - currently has link to view order with incorrect path (profile_order) there as a reminder for the probable user story or eventual adding

### CODE REVIEW

File: app/views/admin/users/show.html.erb [hunk 1]
```diff
@@ -1,27 +1,27 @@
 <h1><%= @user.name %></h1>
-<p>Date Registered: <%= @user.created_at.to_date %></p>
-<p>Enable/Disable: </p>
-<% if @user.enabled? %>
-    <td><%= button_to "Enable", admin_toggle_user_path({user_id: @user.id}) %></td>
-<% else %>
-    <td><%= button_to "Disable", admin_toggle_user_path({user_id: @user.id}) %></td>
-<% end %>
+
```
Comment (tonyr729) [inline app/views/admin/users/show.html.erb]: Does the user story not require Date Registered Im guessing?
